scripts/core/exp-bindless-images.yml

```diff
@@ -32,47 +32,50 @@ etors:
     - name: BINDLESS_IMAGES_SUPPORT_EXP
       value: "0x2000"
       desc: "[$x_bool_t] returns true if the device supports the creation of bindless images"
-    - name: BINDLESS_IMAGES_1D_USM_SUPPORT_EXP
+    - name: BINDLESS_IMAGES_SHARED_USM_SUPPORT
       value: "0x2001"
+      desc: "[$x_bool_t] returns true if the device supports the creation of bindless images backed by shared USM"
```

Comment:
Not sure if this needs spec clarification or if it's just a lack on knowledge on my part.

Is this `BINDLESS_IMAGES_SHARED_USM_SUPPORT` mutually exclusive with `BINDLESS_IMAGES_[1,2]D_USM_SUPPORT_EXP`?

I.e. if I want to create a 2D image backed by shared USM do I need to check both of `BINDLESS_IMAGES_SHARED_USM_SUPPORT` and `BINDLESS_IMAGES_2D_USM_SUPPORT_EXP`?

My initial thought was "why aren't there 1D/2D versions of the shared USM query?" but then I thought maybe that "shared" and "[1,2]D" are independent support properties.

Reply:
> if I want to create a 2D image backed by shared USM do I need to check both of `BINDLESS_IMAGES_SHARED_USM_SUPPORT` and `BINDLESS_IMAGES_2D_USM_SUPPORT_EXP`?

Correct. 

The former (and new proposed) query is for support of bindless images on USM memory accessible by both host and device, as specified by the SYCL 2020 `usm::alloc::shared` type, for example. The latter is for support of 2D bindless images backed by USM, which could be limited to just being accessible on the device/host if supported. 

Does that clarify? I'm not sure I'm explaining it any better here. I would welcome any suggestions to update the `desc` with.

Reply:
Yes, it does clarify and I think it was just a me understanding thing rather than a spec problem.

There could be situations where the image specific USM support flag is required? I.e. support for creating images on shared USM will be different from the presence of shared USM?

Reply:
> support for creating images on shared USM will be different from the presence of shared USM?

@alycm It may be. This would really depend on the adapter in question. Including this flag allows for bindless image support to be stretched to the limits of the adapter being used, rather than, say, having to entirely omit bindless images backed by USM merely because there isn't support for them on shared USM.

Reply:
If it's a hypothetical requirement why just have `SHARED`?  The device properties have two levels of shared plus system.

* `USM_SINGLE_SHARED_SUPPORT`
* `USM_CROSS_SHARED_SUPPORT`
* `USM_SYSTEM_SHARED_SUPPORT`

Reply:
I think your query matches `SYSTEM`.

Reply:
@alycm I've created [691](https://github.com/oneapi-src/unified-runtime/issues/691) in response.

Reply:
Thank you @isaacault !